### PR TITLE
Exposed thing: fixes on Websocket

### PR DESCRIPTION
**Pull request by fillobotto**

### CODE REVIEW

File: packages/binding-websockets/src/ws-server.ts [hunk 1]
```diff
@@ -188,88 +207,45 @@ export default class WebSocketServer implements ProtocolServer {
...
-                            thing.properties[propertyName].forms.push(form);
-                            console.debug(
-                                "[binding-websockets]",
-                                `WebSocketServer on port ${this.getPort()} assigns '${href}' to Property '${propertyName}'`
-                            );
+                    if (!property.writeOnly) {
+                        for (let formIndex = 0; formIndex < thing.properties[propertyName].forms.length; formIndex++) {
+                            thing
+                                .handleObserveProperty(propertyName, observeListener, { formIndex: formIndex })
+                                .then(() => ws.close(0, "Completed"))
```
Comment (relu91) [inline packages/binding-websockets/src/ws-server.ts]: I don't think we need to close the connection after a change is sent to the client. It should be kept open as long as the client likes it.

**Review by relu91 — CHANGES_REQUESTED** [review]
Looks good, I have a concern below.

File: packages/binding-websockets/src/ws-server.ts (hunk 1 above)
Comment (fillobotto) [reply packages/binding-websockets/src/ws-server.ts]: Instead, I should close the connection here, right? https://github.com/relu91/thingweb.node-wot/blob/b3b5fe7a966b10dcd893a27dca56276c7389c73f/packages/binding-websockets/src/ws-server.ts#L242

Comment (relu91) [reply packages/binding-websockets/src/ws-server.ts]: I think that once you get a close event the ws is already closed by the system. In theory, the server should keep the connection open until it shuts down.

**Review by relu91 — APPROVED** [review]
The last commit looks good, but the PR contains 203 File changes. I'll try to change the target branch to master and see how it works.

**Comment by fillobotto** [comment]
> The last commit looks good, but the PR contains 203 File changes. I'll try to change the target branch to master and see how it works.

I merged your master branch into my repo because the previous master had some problems with ws tests and that was the result. It's not the first time...any idea of why this keep happening?

**Comment by relu91** [comment]
> I merged your master branch into my repo because the previous master had some problems with ws tests and that was the result. It's not the first time...any idea of why this keep happening?

Solved by correctly pointing to the master branch 🥳

**Pull request merged**